packages/graphql/src/classes/authorization/parse-request-token.ts

```diff
@@ -17,51 +17,27 @@
  * limitations under the License.
  */
 
-import { IncomingMessage } from "http";
+import type { RequestLike } from "../../types";
+import { DEBUG_AUTH } from "../../constants";
 import Debug from "debug";
-import type { Context } from "../types";
-import { DEBUG_AUTH } from "../constants";
 
 const debug = Debug(DEBUG_AUTH);
 
-type RequestLike = {
-    headers?: { authorization?: string; Authorization?: string };
-    rawHeaders?: Array<string>;
-    cookies?: { token?: string };
-};
-
-export function getToken(context: Context): string | undefined {
-    const req: RequestLike = context instanceof IncomingMessage ? context : context.req || context.request;
-
+export function getToken(req: RequestLike): string | undefined {
```

Comment:
If these two functions aren't used anywhere else (which I don't believe they are), shall we just move them into the `Neo4jGraphQLAuthorization` class as methods?